linux/LinuxMDEparser/README.ME

```diff
@@ -0,0 +1,38 @@
+# LinuxMDEparser
+Microsoft Defender Endpoint Log Parser for Linux and MacOS.
+
+Compiled sources (with pyinstaller --onefile) can be found on Releases.
+
+# About
+The LinuxMDEparser is a tool to export multiple log files from MDE for Linux and Mac to Excel, making it easier to filter and search the logs.
+
+# Help
+You can always check the available options by running: LinuxMDEparser.exe -h
+
+LogFile     Type the filename to convert. Available options:
+
+                 real_time_protection - Troubleshoot performance issues
+                 wdavhistory - Troubleshoot AV acvtivity. You can find this file in var\opt\microsoft\mdatp log folder
+                 real_time_protection.json - Troubleshoot performance issues for Microsoft Defender (https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-atp/linux-support-perf)
+
+To get the real_time_statiscs log file, you can open bash or terminal (MacOS) and just do - mdatp --diagnostic --real-time-protection-statistics > real_time_protection
+
+# Usage
+LinuxMDEparser.exe {log filename}
```

Comment:
This is not .exe on non-Windows.
If you use `LinuxMDEParser {log filename}` then it would be valid on either macOS, Linux, or Windows